examples/single_agent.py

```diff
@@ -50,10 +57,15 @@ def main(scenarios, sim_name, headless, num_episodes, seed, max_episode_steps=No
             action=ActionSpaceType.Continuous,
             max_episode_steps=max_episode_steps,
         ),
+<<<<<<< HEAD
         agent_builder=PPOPolicy,
         agent_params={"policy_params": adapter.policy_params},
         observation_adapter=adapter.observation_adapter,
         reward_adapter=adapter.reward_adapter,
+=======
+        # agent_builder=ChaseViaPointsAgent,
+        agent_builder=PPO,
+>>>>>>> f603a900507e2ba8579b3029d85408c197316b13
```

Comment:
Please remove any merge conflicts from all the files. The above is an example.

Reply:
have some reformating issue. will try to resolve it.

Reply:
removed all the merge conflicts now

Reply:
Hi @Dikshuy, I think there are still a couple more in the CHANGELOG.md... Although, I think the only files that really need to be changed are examples/single_agent.py and ultra/ultra/baselines/adapter.py, is this correct?

Reply:
yeah, only  `examples/single_agent.py` and `ultra/ultra/baselines/adapter.py` need to be changed everything else is the same.

Reply:
Hmm yeah okay. Perhaps you will have to rebase your `smarts_ppo_agent`  branch onto the `smarts_use_ultra_ppo_agent_from_develop` branch and then submit your changes.

Or, you can create a new branch, branching from `smarts_use_ultra_ppo_agent_from_develop`, and then apply the changes and create a new pull request.

Let me know if I anything was confusing or if I can help in any way.